tests/integration_tests/tests/usage_collector_base.py

```diff
@@ -49,9 +47,21 @@ def run_collector_scripts_and_assert(self, messages):
                                join(LOG_PATH, LOG_FILE))
 
     def clean_timestamps(self):
-        # this is necessary for forcing the collector scripts to actually run
+        # This is necessary for forcing the collector scripts to actually run
         # in subsequent tests, despite not enough time passing since last run
         run_postgresql_command(
             self.env.container_id,
-            "UPDATE usage_collector SET hourly_timestamp=0, daily_timestamp=0 "
-            "WHERE id=0")
+            "UPDATE usage_collector SET hourly_timestamp=NULL, "
+            "daily_timestamp=NULL")
+
+    def clean_usage_collector_log(self):
+        # We need to clean the usage_collector log before each test, because
+        # each test uses it for asserting different values.
+        old_usage_log = join(LOG_PATH, self._testMethodName)
+        test_usage_log = join(LOG_PATH, LOG_FILE)
+
+        docker.execute(self.env.container_id,
+                       'mv {0} {1}'.format(test_usage_log, old_usage_log))
```

Comment:
```suggestion
        self.execute_on_manager(['mv', test_usage_log, old_usage_log])
```
ditto for every usage of `docker.execute`. We made the interface on `self` for the tests to use so that they don't need to go to a lower level and know about docker at all (or the container id)

And no need to format it into a string, it will just be split again :P

Reply:
Nice, I'll change it, thanks!